web_favicon/__manifest__.py

```diff
@@ -6,10 +6,7 @@
 {
     "name": "Custom shortcut icon",
     "version": "18.0.1.0.0",
-    "author": "Therp BV, "
-    "Tecnativa, "
-    "OERP Canada,"
-    "Odoo Community Association (OCA)",
+    "author": "Therp BV, Tecnativa, OERP Canada,Odoo Community Association (OCA)",
```

Comment:
There are changes to modules other than the one you've authored in this PR. I assume this is from an overeager pre-commit run against all files? Ideally you'd want this PR only touching/dealing with your new modules.